raco/backends/sql/catalog.py

```diff
@@ -107,10 +107,12 @@ def _convert_zeroary_expr(self, cols, expr, input_scheme):
 
     def _convert_unary_expr(self, cols, expr, input_scheme):
         input = self._convert_expr(cols, expr.input, input_scheme)
-        if isinstance(expr, expression.MAX):
-            return func.max(input)
-        if isinstance(expr, expression.MIN):
-            return func.min(input)
+
+        fname = expr.__class__.__name__.lower()
```

Comment:
We already take advantage of SQLAlchemy's dialects for compiling a SQLAlchemy expression to a SQL string, but I guess this is orthogonal.